daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/NodeInfo.scala

```diff
@@ -473,6 +472,7 @@ object NodeInfo extends Enum {
     }
 
     trait PrimNumeric { self: Numeric.Kind =>
+      val width:MaybeInt
```

Comment:
Generally preferred style for Scala for defining a prototype in a base class/trait is to use def as in 
`def width:MaybeInt`
This can be then implemented as a val, a lazy val, or a def.